pkg/operator/controller/metrics.go

```diff
@@ -0,0 +1,94 @@
+package controller
+
+import (
+	promv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	logc "log"
+	"net/http"
+)
+
+// Handler creates a new prometheus handler to receive scrap requests
+func Handler(MaxRequestsInFlight int) http.Handler {
```

Comment:
It is more a of a MetricHandler or as a comment states MetricsScrapRequest Handler?

Now it looks like a main Handler for anything that `operator/controller/`. I think it is misleading and lacks context.

Reply:
updated